source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
+E.2.3.3 Total_Item_Investigations
+'''''''''''''''''''''''''''''''''
 
-The account used for this testing must have concurrent/simultaneous-user limit set at one single user. A second user attempting to access the database would be denied.
+This test is required when investigations can be reported independently of a request. If all investigations have a matching request, please apply to the COUNTER Project Director for an audit exception prior to the audit commencing.
 
-**Option 1**: The content provider denies the user access when the concurrent/simultaneous-user limit is exceeded upon login.
+Where possible, 50% of items investigated SHOULD be via browsing the platform and 50% via searching the platform. If either browsing or searching is not possible, all items may be investigated via the only available option.
 
-The auditor must force 50 Limit_Exceeded access denials.
+The auditor MUST make a total of 80 investigations against different items, resulting in 80 each of Total_Item_Investigations and Unique_Item_Investigations.
 
-* The auditor will log into the site causing the user limit to reach the maximum allowance. The auditor will then attempt to log into the site using a different computer.
-* The second login should be refused access. Each time access is refused, the auditor will record this as 1 Limit_Exceeded.
+Where a platform has fewer than 80 items, the auditor MUST make at least one investigation per item and testing should result in 80 Total_Item_Investigations and the item-count of Unique_Item_Investigations.
 
-Each of these concurrent/simultaneous turnaways must report 1 Limit_Exceeded in the DR_D2 Standard View.
 
-**Option 2**: The content provider denies the user access when the concurrent/simultaneous user limit is exceeded upon searching or accessing a database.
+E.2.4 Title Report Audit Tests: Books and Reference_Works
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
-The auditor must force 50 Limit_Exceeded turnaways.
+The Title Report will be COUNTER-compliant if the following audit tests are passed. The book-related Standard Views of the Title Report (TR_B1, TR_B2, TR_B3) will be COUNTER-compliant if the metrics match those in the Title Report, with the appropriate aggregation.
 
-* The auditor will log into the site. They will select and make a search on a database or browse to a database causing the user limit to reach the maximum allowance. The auditor will then log into the same site using a different computer. The auditor will then repeat the action made on the previous computer (select and make a search on a database or browse to a database).
-* The user should then be refused access as the concurrent/simultaneous-user limit has exceeded. Each time access is refused, the auditor will record this as 1 Limit_Exceeded.
+Audit test requirements vary depending on the set up of the platform and any related database(s), as indicated by the Options within the tests below.
```

Comment:
"any related database(s)" seems wrong here, maybe "type of content on the platform"?